packages/ember-views/lib/system/build-component-template.js

```diff
@@ -186,6 +211,25 @@ function normalizeComponentAttributes(component, isAngleBracket, attrs) {
     }
   }
 
+  // IE8 Support: IE8 cannot change the type attr of an input after it has been appended to
+  // any node. Therefore, we detect if the browser cannot change the type of the input after
+  // being appended, and unbind type.
+  if (normalized.type && !canChangeType) {
```

Comment:
This seems more complicated than I would have assumed.  

I was thinking something like:

```javascript
if (attrs.type && !canChangeType) {
  attrs.type = getValue(attrs.type); // forces a static value
}
```

Done [around here](https://github.com/emberjs/ember.js/blob/release-1-13/packages/ember-views/lib/system/build-component-template.js#L128) would do the trick.